src/webgpu/util/math.ts

```diff
@@ -62,3 +65,48 @@ export function diffULP(a: number, b: number): number {
   }
   return bits_a + bits_b;
 }
+
+/**
+ * @returns the next single precision floating point value after |val|,
+ * towards +inf if |dir| is true, otherwise towards -inf.
+ */
```

Comment:
Please document the behaviour for -0 and +0
Because they compare as equal, the result is always non-zero.

From IEEE 754-2008:

> nextUp(x) is the least floating-point number in the format of x that compares greater than x.

>  nextUp(±0) is the positive number of least magnitude in x’s format.

Reply:
Done